libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -32,18 +39,11 @@ export class ButtonComponent {
     @Input()
     disabled: boolean;
 
-    /** The type of the button. Types include 'standard', 'positive', 'medium', and 'negative'.
-     * Leave empty for default (Action button).'*/
+    /** The type of the button. Types include 'standard','positive' and 'negative' etc.
```

Comment:
You should not use etc. All types need to be listed here since it's going to the API that the user is reading

Reply:
Removed **etc** and listed all type names.